cobra/manipulation/delete.py

```diff
@@ -15,22 +16,25 @@ def prune_unused_metabolites(cobra_model):
     Parameters
     ----------
     cobra_model: cobra.Model
-        the model to remove unused metabolites from
+    the model to remove unused metabolites from
```

Comment:
This is not consistently done in cobrapy, so I apologize, but since you are touching this docstring can you please use [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) formatting?